app/views/admin/loan_questions/_loan_types.slim

```diff
@@ -0,0 +1,35 @@
+.required-loan-types
```

Comment:
Looks like we're trying to reproduce a simple form field here without using simple form? There should be a way to not do this unless I'm not understanding.